fix: update deployment guide with activation info (#219)
plentymarkets/plentyshop-pwa: docs/deployment/deployment.md

The workflow takes several minutes to complete. You can monitor the progress in the **Actions** menu.

## Activation

1. Log into your plentysystems system.
2. Go to **CMS » Deployment**.

> 3. Click  the three dots under "Actions" on the configured instance
> 4. Click "Activate application" from the menu

Reviewer: ```suggestion
3. For the configured domain, open the further actions menu (three dots).
4. Click on **Activate application**.
```

Reviewer: Would be nice to add another line that describes the expected result.
